packages/xo-web/src/xo-app/backup/restore/restore-metadata-backups-modal-body.js

```diff
@@ -63,30 +68,56 @@ export default class RestoreMetadataBackupModalBody extends Component {
 export class RestoreMetadataBackupsBulkModalBody extends Component {
   static propTypes = {
     nMetadataBackups: PropTypes.number,
+    pools: PropTypes.array,
   }
 
   state = { latest: true }
 
   get value() {
-    return this.state.latest
+    return this.state
   }
 
+  _optionRenderer = ({ timestamp }) => (
+    <FormattedDate
+      value={new Date(timestamp)}
+      month='long'
+      day='numeric'
+      year='numeric'
+      hour='2-digit'
+      minute='2-digit'
+      second='2-digit'
+    />
+  )
+
   render() {
     return (
-      <div>
-        {_('bulkRestoreMetadataBackupMessage', {
-          nMetadataBackups: this.props.nMetadataBackups,
-          oldestOrLatest: (
-            <StateButton
-              disabledLabel={_('oldest')}
-              enabledLabel={_('latest')}
-              handler={this.toggleState('latest')}
-              state={this.state.latest}
-            />
-          ),
-        })}
+      <Container>
+        <Row>
```

Comment:
And you can remove the `<br />` element bellow.
```suggestion
        <Row className="mb-1">
```